cvxpy/reductions/solvers/conic_solvers/mosek_conif.py

```diff
@@ -508,7 +508,11 @@ def invert(self, solver_output, inverse_data):
                 K = inverse_data['K_dir']
                 prim_vars = MOSEK.recover_primal_variables(task, sol_type, K)
                 dual_vars = MOSEK.recover_dual_variables(task, sol_type)
-        attr = {s.SOLVE_TIME: task.getdouinf(mosek.dinfitem.optimizer_time)}
+        attr = {s.SOLVE_TIME: task.getdouinf(mosek.dinfitem.optimizer_time),
+                s.NUM_ITERS: task.getdouinf(mosek.iinfitem.intpnt_iter) +
+                             task.getdouinf(mosek.liinfitem.simplex_iter) +
+                             task.getdouinf(mosek.liinfitem.mio_simplex_iter)
```

Comment:
```suggestion
                s.NUM_ITERS: task.getintinf(mosek.iinfitem.intpnt_iter) +
                             task.getlintinf(mosek.liinfitem.simplex_iter) +
                             task.getlintinf(mosek.liinfitem.mio_intpnt_iter) + 
                             task.getlintinf(mosek.liinfitem.mio_simplex_iter)
```

Haven't tried, but based on the [mosek docs](https://docs.mosek.com/latest/pythonapi/optimizer-task.html#mosek.task.getintinf), I would expect to use `getintinf` for `iinfitem` and `getlintinf` for `liinfitem`.

I have added `mio_intpnt_iter`, I think that's all after searching for `_iter` in the docs.

Reply:
@phschiele thanks! Made changes here: [Add mio interior point](https://github.com/cvxpy/cvxpy/pull/2137/commits/3095d223b24e9144eaec69485a514d050e34076a). 

I could not find other parameters that refer to iterations used until termination. Is a unit test also needed for the attribute ?

Reply:
Yes, you can add a check that the key is present. I would not check the specific number, but `solution[s.NUM_ITERS] >= 0` should work.
Maybe you can add this to an existing test with a similar scope.

Reply:
@phschiele thanks ! I added a new test here: [Add test for number of iterations](https://github.com/cvxpy/cvxpy/pull/2137/commits/4e22b4bc4277ae0685683e27f39465a9f1d2db13). Instead of the new test I could also put an assertion in `test_mosek_params` (but I wasn't sure because the mentioned test refers to solver specific mosek parameters).